newrelic/resource_newrelic_alert_policy_test.go

```diff
@@ -173,12 +174,16 @@ func testAccCheckNewRelicAlertPolicyExists(n string) resource.TestCheckFunc {
 
 func testAccDeleteAlertPolicy(name string) func() {
```

Comment:
Not a blocker, but it looks like we have a duplicate of this function in [helpers_test.go](https://github.com/terraform-providers/terraform-provider-newrelic/pull/358/files#diff-8644a294a323fa7b442332ed8b0d2116R52).